src/client/components/app/ScanNetworkIcon/index.tsx

```diff
@@ -0,0 +1,44 @@
+import styled from 'styled-components';
+import { GeneralVaultView, Network } from '@types';
+import { DefaultscanIcon, EtherscanIcon, FtmscanIcon, Icon } from '@components/common';
+
+interface ScanNetworkIconProps {
+  currentNetwork?: Network;
+  blockExplorerUrl?: string;
+  selectedVault: GeneralVaultView;
```

Comment:
Here you can just pass the 'address' directly, since the rest of the properties of the 'GeneralVaultView' are not needed.